mava/components/updating/parameter_server.py

```diff
@@ -194,6 +215,10 @@ def on_parameter_server_set_parameters(self, server: SystemParameterServer) -> N
                 # # Loop through tuple
                 # for var_i in range(len(server.store.parameters[var_key])):
                 #     server.store.parameters[var_key][var_i].assign(params[var_key][var_i])
+            elif var_key == "best_checkpoint":
```

Comment:
Could we possibly check for a type instead of looking at the var_key to make this more general. Maybe other methods can then also use this.

Reply:
I was thinking of type dict but I ended up making that case happen for the normal network too since it is also a nested and multu dim dict

Reply:
Maybe we can have a chat about this? :)

Reply:
Already updated the function, where we don't need this if statement anymore

Reply:
Great.